rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -73,20 +73,6 @@ pub(crate) struct Managed {
     pub inject_faults: bool,
 }
 
-impl Managed {
-    #[cfg(debug_assertions)]
-    /// In debug mode, if `--inject-faults` is passed, sleep for `millis` milliseconds
-    pub async fn fault_msleep(&self, millis: u64) {
-        if self.inject_faults {
-            tokio::time::sleep(std::time::Duration::from_millis(millis)).await;
-        }
-    }
-
-    #[cfg(not(debug_assertions))]
-    /// Does nothing in release mode
-    pub async fn fault_msleep(&self, _millis: u64) {}
```

Comment:
The fault injection tests weren't used anywhere. I really like the idea but the code is simpler without them. If anyone has a slow hard disk and runs into issues with that, we can deal with it then.